### PR TITLE
Implementing support for p7

### PR DESCRIPTION
**Pull request by rasmus-kirk**
## Purpose

See #61 

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Review by soerenbf — APPROVED** [review]
Looks good 😄

File: concordium-rust-sdk
[the remark this answers is not in the document]
Comment (rasmus-kirk) [reply concordium-rust-sdk:1]: It's already updated, since some of the changes necessary for P7 was already implemented

image

**Comment by soerenbf** [comment]
@rasmus-kirk can we have this merged and the docker image released? I don't know if there's anything else to do to have this "released", but could you find out?

**Comment by rasmus-kirk** [comment]
Yes, I'm on it. Just waiting until I get to work with releasing, but it's my next task. I think there's a Jenkins job for it.

**Pull request merged**